registry/domain/factory/service_factory.py

```diff
@@ -24,3 +26,24 @@ def convert_service_db_model_to_entity_model(service):
             groups=[ServiceGroup(org_uuid=group.org_uuid, service_uuid=group.service_uuid, group_id=group.group_id,
                                  endpoints=group.endpoints, pricing=group.pricing) for group in service.Service.groups]
         )
+
+    @staticmethod
+    def convert_entity_model_to_service_db_model(service):
```

Comment:
Domain layer factory should not contain any reference to repository models. 
Making repo layer objects from domain layer should be done in repo layer
Making domain layer objects from repo layer can sit in the domain as it doesn't refer to repo layer directly it just treats repo layer object as plugging object.